app/routes/docs.client.samples.md

```diff
@@ -164,3 +166,62 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## Working With JSON Schema
+
+GCN Notices are distributed in JSON format. This guide describes how to programmatically read JSON schema.
+
+## Parsing JSON Notices
+
+Read the JSON data from [sample.schema.json](https://gcn.nasa.gov/docs/notices/schema) and [sample.example.json](https://gcn.nasa.gov/docs/notices/schema), which parses it into Python dictionaries.
+
+```python
+import json
+
+# Load and parse schema and example JSON files
+with open('sample.schema.json', 'r') as schema_file:
+    schema = json.load(schema_file)
+
+with open('sample.example.json', 'r') as example_file:
+    example = json.load(example_file)
+
+print('Schema:', schema)
+print('Example:', example)
+```
+
+## Encoding and Decoding of Embedded Data
+
+The following code sample shows how to encode/decode a file in Python. The `base64` package includes the methods `b64decode` and `b64encode` to make this task simple.
+
+```python
+import base64
+
+# Parse the content of your file to a base64 encoded string:
+with open("path/to/your/file", 'rb') as file:
+    encoded_string = base64.b64encode(file.read())
+
+print(encoded_string)
+# b'a1512dabc1b6adb3cd1b6dcb6d4c6......'
+
+# Decode and write the content to a local file:
+with open("path/to/destination/file", 'wb') as file:
+    file.write(base64.b64decode(encoded_string))
+
+```
+
+If you want to include a FITS file in a Notice, you add a property to your schema definition in the following format:
+
+<Highlight
+language="json"
+code={JSON.stringify(
+{
+type: 'string',
+contentEncoding: 'base64',
+contentMediaType: 'image/fits',
+},
+null,
+2
+)}
+/>
+
```

Comment:
@Vidushi-GitHub can you attach a screenshot of this code sample? Does it render correctly in the browser?

Reply:
It was attached above too. 
<img width="662" alt="Screenshot 2024-08-06 at 3 37 44 PM" src="https://github.com/user-attachments/assets/a62ba40e-cbab-4ff3-82fa-071cfa56db8d">